draft-ietf-quic-multipath.md

```diff
@@ -460,9 +469,9 @@ by sending a PATH_ABANDON frame (see {{path-abandon-frame}}) which
 requests the peer to stop sending packets with the corresponding Path Identifier.
 
 When sending or receiving a PATH_ABANDON frame, endpoints SHOULD wait for at
-least three times the current Probe Timeout (PTO) interval as defined in
-{{Section 6.2 of QUIC-RECOVERY}} after the last packet was sent on the path,
-before sending the MP_RETIRE_CONNECTION_ID frame for all the corresponding Connection
+least three times the current Probe Timeout (PTO) interval after the last
+packet was sent on the path, as defined in {{Section 6.2 of QUIC-RECOVERY}},
+before sending the MP_RETIRE_CONNECTION_ID frame for all the corresponding connection
 IDs used for this path. This is inline with the requirement of {{Section 10.2 of QUIC-TRANSPORT}}
```

Comment:
This sentence is really unclear. It says "for all corresponding CIDs used for this used". However, there is only one CID in use at a time. 

Is this meant to mean that you should send multiple MP_RETIRE_CONNECTION_ID frames for all CIDs that have previously been issue for this path ID or only one frame for the one that is currently actively used?

If the former we should change the MP_RETIRE_CONNECTION_ID frame to be able to carry multiple CIDs.

Reply:
Actually endpoints do have chance to announce more than one unused CIDs for the corresponding path. 
This mechanism try to explain: Endpoints need to send MP_RETIRE_CONNECTION_ID frames for each CID related to the Path ID, but only need one MP_RETIRE_CONNECTION_ID frame for each CID.

I don't think we need to change the MP_RETIRE_CONNECTION_ID frame to be able to carry multiple CIDs, as in this part we try to keep the same habit with RFC9000, which only changes applied on each path. Endpoints could always put several MP_RETIRE_CONNECTION_ID frames in the same packet, then it will not affect the performance.

Reply:
We anyway create a new frame. In RFC9000, there is usually not a need to retire multiple CIDs at once. However, when a path is abandoned, this can happen often with. Therefore we should optimise the MP_RETIRE_CONNECTION_ID for that case.

However, I can open a separate issue of that. For now I propose the editorial change below to make it more clear:

```suggestion
before sending MP_RETIRE_CONNECTION_ID frames.
This is inline with the requirement of {{Section 10.2 of QUIC-TRANSPORT}}
Both endpoints SHOULD send MP_RETIRE_CONNECTION_ID frames
for all connection IDs associated to the Path ID of the abandoned path.
```